internal/services/keyvault/key_vault_certificates_data_source_test.go

```diff
@@ -0,0 +1,83 @@
+package keyvault_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-provider-azurerm/internal/acceptance"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/acceptance/check"
+)
+
+type KeyVaultCertificatesDataSource struct{}
+
+func TestAccDataSourceKeyVaultCertificates_basic(t *testing.T) {
+	data := acceptance.BuildTestData(t, "data.azurerm_key_vault_certificates", "test")
+	r := KeyVaultCertificatesDataSource{}
+
+	data.DataSourceTest(t, []acceptance.TestStep{
+		{
+			Config: r.basic(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).Key("names.#").HasValue("9"),
```

Comment:
```suggestion
				check.That(data.ResourceName).Key("names.#").HasValue("31"),
```

Reply:
i can't edit this PR, @harshavmb once this change is made this is good to go

Reply:
Hi @katbyte ,

I've pushed the change from my end.